diff -Naur A/Makefile B/Makefile
--- A/Makefile	1970-01-01 00:00:00.000000000 +0000
+++ B/Makefile	1970-01-01 00:00:00.000000000 +0000
@@ -1313,4 +1313,6 @@
 	+$(Q)$(MAKE) -f $(srctree)/Makefile olddefconfig
 
+dtbs_install:
+	$(Q)mkdir -p $(INSTALL_DTBS_PATH); cp $(buildRoot)/arch/arm64/boot/dts/tegra210-odin.dtb $(buildRoot)/arch/arm64/boot/dts/tegra210b01-fric.dtb $(buildRoot)/arch/arm64/boot/dts/tegra210b01-odin.dtb $(buildRoot)/arch/arm64/boot/dts/tegra210b01-vali.dtb $(INSTALL_DTBS_PATH)/
 # ---------------------------------------------------------------------------
 # Modules
